.github/workflows/build_wheels_linux.yml

```diff
@@ -65,6 +65,11 @@ on:
         required: false
         type: boolean
         default: true
+      build-command:
```

Comment:
Perhaps instead of passing build-command we should be pass variable like:
```
use-python-build-package: 
  default: true
```

Previous PR contained clean step:
https://github.com/pytorch/test-infra/pull/5071/files

like this
```
${CONDA_RUN} python setup.py clean
```

We should bring it back when use-python-build-package is false. 

Does ``python -m build --wheel -n`` has clean command ?

Reply:
@joecummings sounds good. For vision and audio nightly jobs where successful and wheels seems to working fine. However more tests are required.

Reply:
@joecummings One more idea for experimentation

Rather then boolean do:
```
build-platform with values "python-build-package", "setup-py" and "cibuildwheel"
```
With setup-py being a default. This will allow best flexibility for building wheels.

We want to try to use following package for our builds next:
https://github.com/pypa/cibuildwheel

Reply:
@atalman Got it

```
if inputs.build-platform == 'python-build-package':
    python -m build 
elif inputs.build-platform == 'setup-py;:
    python setup.py bdist
elif inputs.build-platform == 'cibuildwheel':
    OTHER_COMMAND
```

What would this OTHER_COMMAND be?